README.md

```diff
@@ -16,6 +16,23 @@ commonPassworList.test('password');
 commonPasswordList.test('@!#^GDSAQ@#^Q#@^$YAESFDAS');
 ```
 
+## Tagging a release
+
+Create a version branch:
+> git checkout -b &lt;<version>&gt;
+
+Update package.json, package-lock.json,
+CHANGELOG.md, and create a tag:
+> npm version &lt;<version>&gt;
```

Comment:
Maybe it's self-evident, but should the instructions say to commit at this point? Or does the script do the commit?

Reply:
I had an inner Phil telling me that if I have to list these steps, they should be automated. Working on an update now.